locales/en/messages.json

```diff
@@ -1993,6 +1990,9 @@
     "filterWarning": {
         "message": "<span class=\"message-negative\"><b>Warning:</b></span> The amount of filtering you are using is dangerously low. This is likely to make the craft hard to control, and can result in flyaways. It is highly recommended that you <b>enable at least one of Gyro Dynamic Lowpass or Gyro Lowpass 1 and at least one of D-Term Dynamic Lowpass or D Term Lowpass 1</b>."
     },
+    "receiverHelp": {
+        "message": "<b>Basic setup</b>: For serial connections, go to the Ports tab and enable ‘Serial Rx’ for the correct UART.  In the Receivers tab, choose ‘Serial based receiver…’, and set the correct link/data format at the ‘Serial Rx Provider’ drop-down.  For other types of receiver, set the correct mode (ppm, pwm etc).  Choose the correct ‘Channel Map’ for your radio.  Bind the receiver and check that the Roll, Pitch and other channel bar graphs move correctly with stick movement.  You may need to reconfigure Map, Mode, Channels or Mix so that, for example, moving the Roll stick changes Roll values.  Adjust the channel endpoint or range values in the transmitter so that all channels go from ~1000 to ~2000.  Trim the midpoint to 1500.  RC Smoothing is best left at defaults.  For more information, please read receiver chapter of the <strong><a href=\"https://github.com/betaflight/betaflight/blob/master/docs/Rx.md\" target=\"_blank\" rel=\"noopener noreferrer\">documentation</a></strong>.<br /></br /><span class=\"message-negative\"><strong>Important</strong></span>:<br /> &#8226; Don't forget to configure and check failsafe in the Failsafe tab!<br /> &#8226; It is highly recommended to use the latest Tx firmware.  OpenTx or EdgeTx users should disable the hardware> ADC filter with Betaflight.  For more information see <strong><a href=\"https://github.com/betaflight/betaflight/blob/master/docs/Rx.md#disable-adc-filter\" target=\"_blank\" rel=\"noopener noreferrer\">documentation</a></strong>"
```

Comment:
disable adc filter url is wrong, should be https://github.com/betaflight/betaflight/blob/master/docs/Rx.md#disabling-the-opentxedgetx-adc-filter